manifests/art.yaml

```diff
@@ -1,5 +1,5 @@
 updates:
-  - file: "{MAJOR}.{MINOR}/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml" # relative to this file
+  - file: "stable/manifests/kubernetes-nmstate-operator.clusterserviceversion.yaml" # relative to this file
```

Comment:
If we're going to move from version-specific directories to stable then the directory needs to be renamed too. It looks like that is not happening in this PR right now.

Reply:
Looks to me it is: https://github.com/openshift/kubernetes-nmstate/pull/315/files#diff-705a95d35966128e600ba98080c46c11e9c384b996e43a8e64823df68e18eecd